src/main/java/cn/nukkit/item/Item.java

```diff
@@ -33,18 +33,19 @@
 public class Item implements Cloneable, BlockID, ItemID {
     //Normal Item IDs
 
-    protected static String UNKNOWN_STR = "Unknown";
+    private static final ArrayList<Item> creative = new ArrayList<>();
     public static Class[] list = null;
-
-    protected Block block = null;
+    public static Class[] customblocklist = new Class[65535];
```

Comment:
this is initialized later in `init()`.

Reply:
I also agree, but I think this would benefit the community in the short and medium turn.